src/styles/_register.scss

```
@@ -0,0 +1,214 @@
+$mobile: 300px;
+$ipad: 768px;
+@mixin mobile {
+  @media (min-width: #{$mobile})  and (max-width : $ipad) {
+    @content;
+  }
+}
+@mixin ipad {
+  @media (min-width: #{$ipad}) {
+    @content;
+  }
+}
+@mixin firefox-only {
+  @at-root {
+    @-moz-document url-prefix() {
+      & {
+        @content;
+      }
+    }
+  }
+}
+
+.auth {
+  display: flex;
+  flex-flow: row wrap;
+  padding: 10px 15px;
+  background: url("../assets/curved.jpeg") no-repeat;
+  background-size: cover;
+  background-position: center center;
+
+  .side {
+    flex-basis: 50%;
+    padding-left: 55px;
+    display: block;
+    height: 520px;
+    padding-top: 20px;
+    color: #fff;
```

Add this margin to bring is done a little
```suggestion
    color: #fff;
    margin-top: 100px;
```